sashimi-webapp/src/components/Editor.vue

```diff
@@ -23,6 +23,7 @@ import wrapper from '../logic/wrapper';
 import navbar from './Navbar';
 
 let content = '';
+/* eslint : 0 */
```

Comment:
It will be better to explain why this rule switched off for this instance.

e.g. Add a `//TODO: To be sent to logic component`